src/pages/reference/transforms/naming-convention.md

```diff
@@ -48,6 +48,21 @@ For information about `bare` and `wrap` modes, please read the [dedicated sectio
 
 ## Config API Reference
 
+<InlineAlert variant="info" slots="text"/>
+
+<p>
+The following case options should not be used because they violate the <a href="https://spec.graphql.org/October2021/#sec-Names)">GraphQL spec</a>
+
+- `capitalCase`
+- `dotCase`
+- `headerCase`
+- `noCase`
+- `paramCase`
+- `pathCase`
+- `sentenceCase`
+
+</p>
+
 -  `mode` (type: String (`bare` | `wrap`)) - Specify to apply naming-convention transforms to bare schema or by wrapping original schema
 -  `typeNames` (type: String (`camelCase` | `capitalCase` | `constantCase` | `dotCase` | `headerCase` | `noCase` | `paramCase` | `pascalCase` | `pathCase` | `sentenceCase` | `snakeCase` | `upperCase` | `lowerCase`))
 -  `fieldNames` (type: String (`camelCase` | `capitalCase` | `constantCase` | `dotCase` | `headerCase` | `noCase` | `paramCase` | `pascalCase` | `pathCase` | `sentenceCase` | `snakeCase` | `upperCase` | `lowerCase`))
```

Comment:
Can we add another note about deprecating those options along with the suggestion that they should not be used?

We ll deprecate them soon, in a diff ticket, so we might as well mention it in the docs till then.

Reply:
@revanth0212 This note occurs directly before the configuration options in order to preempt all of them instead of adding a note after each option.

Reply:
I think @revanth0212 means mentioning that "not only should you not use these, but they will eventually be unavailable entirely"